src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php

```diff
@@ -23,21 +23,21 @@
 class SodiumVault extends AbstractVault implements EnvVarLoaderInterface
 {
     private ?string $encryptionKey = null;
-    private string|\Stringable|null $decryptionKey = null;
     private string $pathPrefix;
     private ?string $secretsDir;
-    private ?string $derivedSecretEnvVar;
 
     /**
-     * @param $decryptionKey A string or a stringable object that defines the private key to use to decrypt the vault
-     *                       or null to store generated keys in the provided $secretsDir
+     * @param string|\Stringable|null $decryptionKey string or a stringable object that defines the private key to use to decrypt the vault
+     *                                               or null to store generated keys in the provided $secretsDir
      */
-    public function __construct(string $secretsDir, #[\SensitiveParameter] string|\Stringable|null $decryptionKey = null, ?string $derivedSecretEnvVar = null)
-    {
+    public function __construct(
+        string $secretsDir,
+        #[\SensitiveParameter]
+        private string|\Stringable|null $decryptionKey = null,
```

Comment:
the attribute should be on the same line

Reply:
wasn't sure about this one, noted for the next time. Thanks!